gettsim/taxes/calc_taxes.py

```diff
@@ -33,27 +33,25 @@ def tax_sched(
     tax_unit["abgst_tu"] = tax_unit["abgst"]
     tax_unit.loc[adult_married, "abgst_tu"] = tax_unit["abgst"][adult_married].sum()
 
-    """Solidarity Surcharge. on top of the income tax.
-    No Soli if income tax due is below € 920 (solifreigrenze)
-    Then it increases with 0.2 marginal rate until 5.5% (solisatz)
-    of the incometax is reached.
-    As opposed to the 'standard' income tax,
-    child allowance is always deducted for soli calculation
-    There is also Soli on capital income tax, but always with 5.5%. (§3 (3) S.2 SolzG 1995)
+    tax_unit = calc_soli(tax_unit, soli_st_params)
+
+    return tax_unit
+
+
+def calc_soli(tax_unit, params):
+    """Solidarity Surcharge.
+
+    Solidaritätszuschlaggesetz (SolZG) in 1991 and 1992.
+    Solidaritätszuschlaggesetz 1995 (SolZG 1995) since 1995.
     """
```

Comment:
maybe a few bits from the old docstring.
- what is the soli?
- how does it work?
- what is the tax base?

Reply:
Not all functions are tested, too.